lib/y2_logs_helper.pm

```diff
@@ -115,7 +116,13 @@ sub verify_license_translations {
         }
         wait_screen_change { type_string(substr($lang, 0, 1)) } unless (check_var('VIDEOMODE', 'text'));
         send_key_until_needlematch("license-language-selected-dropbox-$lang", 'down', 60);
-        send_key 'ret';
+        if (is_s390x()) {
```

Comment:
Is that module scheduled anywhere else? Is it reproducible on zVM too?

Reply:
The module is scheduled. The part with keyboard selection is only in those jobs, which have INSTALLER_EXTENDED_TEST. As I remember, we decided to leave it only for installed-extended job.

It is not running on zVM.

Reply:
Yep, looks like it's used in the single scenario: https://openqa.suse.de/tests/overview?test=installer_extended